tests/unit/common.py

```diff
@@ -23,7 +23,7 @@
 from _pytest.fixtures import FixtureLookupError, FixtureFunctionMarker
 
 # Worker timeout for tests that hang
-DEEPSPEED_TEST_TIMEOUT = 600
+DEEPSPEED_TEST_TIMEOUT = int(os.environ.get('DEEPSPEED_TEST_TIMEOUT', '600'))
```

Comment:
I don't see it in training unit tests, but in inference tests, I notice that model download and some tests are time consuming. To prevent any timeout issues, I have made this change.

Reply:
I see, could we rename it to `DS_UNITTEST_TIMEOUT` to match the convention on other environment variables?

Reply:
Added changes as per above.

Reply:
Thanks!